rewrite-gradle/src/test/java/org/openrewrite/gradle/gradle8/JacocoReportDeprecationsTest.java

```diff
@@ -458,4 +459,199 @@ void alsoWorkOnKotlinDsl() {
           )
         );
     }
+
+    @Nested
+    class AsMethodInvocation {
+        @Test
+        void deprecationsInMethodInvocationSyntax() {
+            rewriteRun(
+              buildGradle(
+                """
+                  plugins {
+                      id "java"
+                      id "jacoco"
+                  }
+
+                  jacocoTestReport {
```

Comment:
We should probably cover all possible notations available to Groovy here.

```
jacocoTestReport {
    reports {
       xml.enabled = true
       xml.setEnabled(true)
       xml.setEnabled true
       xml.enabled(true)
       xml.enabled true
    }
}
```

It would probably make sense long-term to create a trait for parsing property value assignments for the Groovy and Kotlin DSL.